src/_nebari/stages/kubernetes_keycloak/template/modules/kubernetes/keycloak-helm/values.yaml

```
@@ -33,7 +33,7 @@ extraInitContainers: |
           chown 1000:1000 /data/keycloak-metrics-spi-2.5.3.jar &&
           chmod 777 /data/keycloak-metrics-spi-2.5.3.jar
       fi
-    image: busybox:1.31
+    image: busybox:1.36.1
```

```suggestion
    image: busybox:1.36
```
Let's pin to the minor version rather than the patch